lib/theme/widgets/hs_warning_bottom_sheet.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+class HsWarningBottomSheetView extends StatelessWidget {
+  // final ValueChanged value;
+  final String? iconUrl;
+  final String? title;
+  final String? description;
+  final String? primaryButtonLabel; // return false value
+  final VoidCallback? primaryOnPressed;
+  final String? secondaryButtonLabel; // return true value
+  final VoidCallback? secondaryOnPressed;
+  final String? textButtonLabel;
+  final VoidCallback? textButtonOnPressed;
+  final String? tertiaryButtonLabel;
+  final VoidCallback? tertiaryButtonOnPressed;
+  const HsWarningBottomSheetView({
+    super.key,
+    // required this.value,
+    this.iconUrl,
+    this.title,
+    this.description,
+    this.primaryButtonLabel,
+    this.primaryOnPressed,
+    this.secondaryButtonLabel,
+    this.secondaryOnPressed,
+    this.textButtonLabel,
+    this.textButtonOnPressed,
+    this.tertiaryButtonLabel,
+    this.tertiaryButtonOnPressed,
+  });
+  @override
+  Widget build(BuildContext context) {
+    List<Widget> modalContent = [];
+    List<Widget> buttonGroup = [];
+    Widget button;
+    // Icon configuration
+    modalContent = [
+      Padding(
+          padding: const EdgeInsets.only(bottom: HsDimens.spacing4),
+          child: Text(title ?? 'Title',
```

Comment:
if title != null then show the title, don't show the default title.

Reply:
already removed